apps/web/package.json

```diff
@@ -6,10 +6,12 @@
     "start": "pnpm panda --watch & cross-env NODE_OPTIONS=--max_old_space_size=8192 DISABLE_ESLINT_PLUGIN=true PORT=4200 react-app-rewired start",
     "prebuild": "rimraf build",
     "build": "pnpm panda && cross-env NODE_OPTIONS=--max_old_space_size=4096 GENERATE_SOURCEMAP=false DISABLE_ESLINT_PLUGIN=true react-app-rewired --max_old_space_size=4096 build",
+    "build:web-for-dashboard": "pnpm panda && cross-env NODE_OPTIONS=--max_old_space_size=4096 GENERATE_SOURCEMAP=false DISABLE_ESLINT_PLUGIN=true PUBLIC_URL=/legacy/ react-app-rewired --max_old_space_size=4096 build && pnpm envsetup:dashboard",
```

Comment:
The difference from the normal build is that we use `PUBLIC_URL=/legacy/` variable which will build the `index.html` file with all the resources URLs starting with the `/legacy` prefix.

Reply:
Then, after the build, it also runs the `envsetup: dashboard,` which creates the `env-config.js` file.